Lib/test/test_http_cookiejar.py

```diff
@@ -123,6 +123,11 @@ def test_http2time_garbage(self):
                               "http2time(%s) is not None\n"
                               "http2time(test) %s" % (test, http2time(test)))
 
+    def test_http2time_redos_regression_actually_completes(self):
+        # LOOSE_HTTP_DATE_RE was vulnerable to malicious input which caused catastrophic backtracking (REDoS).
+        # If we regress, this test will take a very long time to succeed.
+        http2time("1-c-1{}!".format(" " * 65506))
```

Comment:
It does not look like a time. I suggest something like:
```python
http2time('09 Feb 1994{}22:23:32 GMT!'.format(' '*10**5))
http2time('09 Feb 1994 22:23:32{}GMT!'.format(' '*10**5))
```
And for `iso2time()`:
```python
iso2time('1994-02-03{}14:15:29 -0100!'.format(' '*10**6))
iso2time('1994-02-03 14:15:29{}-0100!'.format(' '*10**6))
```

Reply:
Nice